### PR TITLE
(DOCSP-29672) JWT exp does not control refresh token expiration

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-29672) JWT exp does not control refresh token expiration

### Staged Changes

- [Authenticate & Manage Users > Manage User Sessions](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/jwt-exp/users/sessions/)
- [Authenticate & Manage Users > Custom JWT Auth](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/jwt-exp/authentication/custom-jwt/)

### CODE REVIEW

File: source/authentication/custom-jwt.txt [hunk 1]
```diff
@@ -888,12 +888,15 @@ table explains each field and the value expected:
...
-     - The current date-time in seconds since Unix epoch. 
+     - "Issued At" is a Unix epoch timestamp of when the token was issued.
+     - The current date-time in seconds since Unix epoch.
    
    * - ``exp``
-     - The date-time when this token will expire. 
-     - A future date-time (in seconds since Unix epoch) for when you want the token to expire. Depending on your business needs, this can be any duration.
+     - The date-time when this token expires.
+     - A specific time defined in seconds since January 1, 1970 (i.e.
+       the Unix epoch). Before this time the JWT is valid. After this
```
Comment (dacharyc) [inline source/authentication/custom-jwt.txt:897]:
```suggestion
       the Unix epoch). Before this time, the JWT is valid. After this
```

File: source/users/sessions.txt [hunk 2]
```diff
@@ -310,13 +310,10 @@ Configure Refresh Token Expiration
...
 configure this time for your App's refresh tokens to be anywhere between
-30 minutes and 180 days. There are two exceptions:
+30 minutes and 180 days.
 
-- Anonymous user refresh tokens do not expire, though anonymous user
-  accounts are deleted after 90 days of inactivity.
-
-- Custom JWT refresh token expiration is determined by the ``exp`` value
-  of the user's JWT.
+Anonymous user refresh tokens do not expire. However, Anonymous user
```
Comment (dacharyc) [inline source/users/sessions.txt:315]: Not blocking for this review, but I wonder if this is still true, or if a configured refresh token expiration (or even the default refresh token expiration) overrides this?

**Review by dacharyc — APPROVED** [review]
One grammar nit, but otherwise LGTM. Although now I have a non-blocking big picture Q about something we touch here.

File: source/users/sessions.txt (hunk 2 above)
Comment (nlarew) [reply source/users/sessions.txt]: Checked with eng - still true but we had a slight error here anyway that I fixed

**Pull request merged**